security/fma-u16a.md

```diff
@@ -0,0 +1,171 @@
+
+#### Description
+
+Chains may incorrectly configure the system-level feature toggles introduced in U16a. This includes enabling ETHLockbox flag without having the ETHLockbox contract deployed, failing to enable ETHLockbox when it's needed, or setting arbitrary unsupported feature flags.
+
+#### Risk Assessment
+
+- Impact: LOW/MEDIUM
+  - Reasoning: Most misconfigurations have no functional effect or cause temporary bridge issues that can be resolved by admin intervention. The most impactful scenario is failing to enable ETHLockbox when needed, which temporarily breaks the bridge.
+- Likelihood: LOW/MEDIUM
+  - Reasoning: Feature toggle system is new, increasing chance of operator error. However, validator checks and clear documentation should minimize risks.
+
+#### Mitigations
+
+- ETHLockbox can only be enabled if lockbox address is set (requires U16 upgrade path).
+- Admin can set toggle post-upgrade if initially misconfigured.
+- Clear operational documentation and runbooks.
+
+#### Detection
+
+- Monitor for portal/lockbox balances.
+- Monitor for unexpected feature flag changes.
+
+#### Action Items
+
+- [ ] Update standard validator to verify lockbox/feature flag configuartion
+- [ ] Set up system feature flag monitor
+- [ ] Set up portal/lockbox balance monitor
+
+#### Recovery Path(s)
+
+- Correct misconfigured toggles via admin transactions.
+- No special recovery procedures beyond standard admin intervention.
+
+### FM2: ETHLockbox Migration Gap
+
+#### Description
+
+Chains upgrading from U15 directly to U16a cannot adopt ETHLockbox until a later upgrade. This creates operational complexity as some chains have ETHLockbox while others don't, leading to testing asymmetry and incident response divergence in pause mechanism runbooks.
+
+#### Risk Assessment
+
+- Impact: LOW
+  - Reasoning: Operational overhead is minimal and doesn't affect core functionality or safety. Primarily creates complexity in testing and operational procedures.
+- Likelihood: HIGH
+  - Reasoning: This is an intentional design decision that will definitely occur for chains following the U15 => U16a upgrade path.
+
+#### Mitigations
+
+- Testing covers both scenarios since major chains exist in both categories.
+- Runbooks and supporting documentation will be updated to prevent operator confusion.
+- Clear documentation of which chains have ETHLockbox and which don't.
+
+#### Detection
+
+- Track which chains follow which upgrade path.
+- Monitor for operator confusion in incident response procedures.
+
+#### Action Items
+
+- [ ] Update incident response runbooks to handle both ETHLockbox and non-ETHLockbox chains
+- [x] Ensure test coverage includes both scenarios
+- [ ] Do a training session about this new state of affairs
+
+#### Recovery Path(s)
+
+- No recovery needed as this is expected behavior.
+- Chains can adopt ETHLockbox in future upgrades.
+- Operational procedures handle both pathways as designed.
+
+### FM3: OPContractsManager Dual-Path Upgrades
```

Comment:
I wonder if we also should have some kind of "tech debt" failure mode, we have a lot of branches now in various parts of the codebase for OptimismPortalInterop, and if we don't get that back into the mainline the complexity could compound as we add new features